### PR TITLE
Revamp Block Explorers Page

### PR DESCRIPTION
**Pull request by themacexpert**
This PR primarily includes the following revisions to the Block Explorers page:
- Add Moonscan documentation
- Update all images and links for to reflect Moonriver by default
- Add Quick Links section for Moonriver and Moonbase Alpha including hyperlinks for the block explorers and full URLs for copy / paste purposes. 

This PR also includes:
- Increase collator count from 52 to 58 for Moonriver.
- New Release tag versions and sha256 hashes
- Update the Faucet page to reflect our renamed faucet bot.

### CODE REVIEW

**Review by albertov19 — CHANGES_REQUESTED** [review]
I pushed a commit adding back Expedition. I think it is important we keep it because it is all JSON RPC based which is nice, and also it provides a basic explorer for a Moonbeam Dev Node. Can you add back the section and update the image?

Thanks

File: builders/tools/explorers.md [hunk 1]
```diff
@@ -61,22 +61,34 @@ As main features, Blockscout offers:
...
 ![Blockscout Explorer](/images/builders/tools/explorers/explorers-2.png)
 
+### Expedition {: #expedition } 
+
+A Moonbeam-themed version of the [Expedition](https://github.com/xops/expedition){target=_blank} explorer can be found in [this link](https://moonbeam-explorer.netlify.app/){target=_blank}.
+
+By default, the explorer is connected to the Moonriver. However, you can switch to another network or connect it to a local dev node by following the next steps:
+
+ 1. Click on the gear icon in the top right corner
+ 2. Select "Development" if you have a node running on `{{ networks.development.rpc_url }}` (default RPC location of a Moonbeam node running with `--dev`flag). You can also switch back to "Moonbase Alpha"
```
Comment (eshaben) [inline builders/tools/explorers.md:71]: doing a drive-by haha sorry... I've been slowly trying to convert to using bold fonts for UI items across the docs site... so like instead of "Development" it would be **Development**

Comment (eshaben) [reply builders/tools/explorers.md:71]: taking some things from the google documentation style guide 🙂  https://developers.google.com/style/text-formatting

**Review by albertov19 — APPROVED** [review]
LGTM!
